File: app/src/main/java/com/dxm/aimodel/modules/model/entity/HomeItem.java
package com.dxm.aimodel.modules.model.entity;

public class HomeItem {
    public String name;
    public String desc;
    public String icon;

    public int id;

    public HomeItem(String name, String desc, int id) {
        this(name, desc, "", id);
    }

    public HomeItem(String name, String desc, String icon, int id) {
        this.name = name;
        this.desc = desc;
        this.icon = icon;
        this.id = id;
    }
}

File: app/src/main/java/com/dx/health/model/entity/HomeItem.kt
package com.dx.health.model.entity

/**
 * Author: Meng
 * Date: 2024/11/21
 * Modify: 2024/11/21
 * Desc:
 */
class HomeItem(
    var name: String?, var desc: String?, var icon: String,
    var id: Int
) {
    constructor(name: String?, desc: String?, id: Int) : this(name, desc, "", id)
}
